crates/bevy_diagnostic/src/lib.rs

```diff
@@ -18,3 +18,7 @@ impl Plugin for DiagnosticsPlugin {
         app.init_resource::<Diagnostics>();
     }
 }
+
+/// The width which diagnostic names will be printed as
+/// Plugin names should not be longer than this value
+pub const MAX_DIAGNOSTIC_NAME_WIDTH: usize = 32;
```

Comment:
What's the purpose of having this limit in the first place?

Reply:
it came in #1505

Reply:
it was to clean up how it's displayed, and it's an issue only for this diagnostics as all other have a fixed name